azure-pipelines.yml

```
@@ -110,6 +110,7 @@ jobs:
     inputs:
       contents: |
         publishManifest.json
+        Shared\publishManifest2022.json
```

Maybe put it into different folder

```suggestion
        VsixManifests\publishManifest2019.json
        VsixManifests\publishManifest2022.json
```